data/include/data/RobotParameters.h

```diff
@@ -0,0 +1,29 @@
+//
+// Created by rolf on 26-10-20.
+//
+
+#ifndef RTT_ROBOTPARAMETERS_H
+#define RTT_ROBOTPARAMETERS_H
+
+#include <roboteam_proto/RobotParameters.pb.h>
+
+class RobotParameters {
+public:
+
+    RobotParameters();
+    RobotParameters(double radius, double height, double frontWidth, double dribblerWidth, double angleOffset);
+    explicit RobotParameters(const proto::RobotParameters& protoParams);
+    [[nodiscard]] proto::RobotParameters toProto() const;
+    static RobotParameters DEFAULT();
+    static RobotParameters RTT_2020();
```

Comment:
Uhmmmmm, you should learn Rust you'll like it.

```suggestion
    static RobotParameters from_default();
    static RobotParameters from_rtt2020();
```